_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -57957,7 +57945,7 @@
 /turf/open/floor/plating,
 /area/maintenance/starboard/aft)
 "cOB" = (
-/obj/item/seeds/sunflower/moonflower,
```

Comment:
boy oh boy I sure do hate those people who try to have fun in maintenance by growing moonshine

Reply:
mutated plant it has no place as a static maint spawn

Reply:
but it's a _garbage_ mutated plant.

also, geraniums and lilies are mutated plants, and don't they spawn in some chapels?